Packs/Signum/Integrations/Signum/Signum.py

```diff
@@ -0,0 +1,297 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+""" IMPORTS """
+# Std imports
+import os
+from datetime import datetime
+
+# 3-rd party imports
+import urllib3
+import base64
+import hashlib
+from xml.etree import ElementTree
+import re
+"""
+
+GLOBALS/PARAMS
+
+Attributes:
+    INTEGRATION_NAME:
+        Name of the integration as shown in the integration UI, for example: Microsoft Graph User.
+
+    INTEGRATION_COMMAND_NAME:
+        Command names should be written in all lower-case letters,
+        and each word separated with a hyphen, for example: msgraph-user.
+
+    INTEGRATION_CONTEXT_NAME:
+        Context output names should be written in camel case, for example: MSGraphUser.
+"""
+
+
+INTEGRATION_NAME = 'Signum'
+INTEGRATION_COMMAND_NAME = 'signum'
+INTEGRATION_CONTEXT_NAME = 'Signum'
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    def test_module(self, username: str, password: str) -> dict:
+        """
+            Performs basic request to check if the API is reachable and authentication is successful.
+        Returns:
+            Response dictionary
+        """
+        return self.list_domain_users(domain_id=1, username=username, password=password)
+
+    def list_domain_users(self, domain_id: int, username: str, password: str) -> dict:
```

Comment:
The client class already maintains the `username` and `password` (in self.auth) - why not use it?